### DISCUSSION TITLE
How can I get a Brain Galaxiy Acievment.

### DISCUSSION THREAD

**Discussion by iamtusharahmed** [community/community#75885] [New to GitHub]
I saw some tutorial, I its not working

**Comment by iambashir** [comment]
You can search it on youtube

**Reply by iamtusharahmed** [reply]
Sure

**Comment by iamtusharahmed** [comment]
ok

**Reply by iambashir** [reply]
You can follow the link https://www.youtube.com/watch?v=oJWFfp3sGYA&t=839s

**Comment by iamtusharahmed** [comment]
> You can follow the link https://www.youtube.com/watch?v=oJWFfp3sGYA&t=839s
Thanks

**Comment by iamtusharahmed** [comment]
Thanks
